src/client/stylesheets/style.less

```diff
@@ -435,13 +441,6 @@ hr.wide {
 		padding-bottom: 0.3em;
 	}
 	.note-content {
-		quotes: "\201C" "\201D";
```

Comment:
This removal is part of another PR if I'm not mistaken.

Reply:
@MonkeyDo I have reverted this  and added the `text-decoration` css property , let me  know if there's any changes needed :)